components/InteroperablePlanets.vue

```diff
@@ -104,7 +104,7 @@
   &__sun &__back
     width 180%
     height 180%
-    background radial-gradient(50% 50% at 50% 50%, rgba(0, 71, 255, 0.8) 56%, rgba(0, 41, 255, 0.38) 70%, rgba(0, 30, 108, 0) 100%)
+    // background radial-gradient(50% 50% at 50% 50%, rgba(0, 71, 255, 0.8) 56%, rgba(0, 41, 255, 0.38) 70%, rgba(0, 30, 108, 0) 100%)
```

Comment:
Let's revert to using a box-shadow on `planet__front` and see if it holds up:
```
box-shadow 0px 0px 3.5em #2138ff, 0px 0px 8em #0057FF
```
We may need to add a 3d transform hack to the planet to avoid glitches. Also we won't need `.planet__back` in the sun element anymore, since it doesn't do anything anymore.